layers/meta-balena-imx8m-var-dart/recipes-bsp/u-boot/u-boot-variscite.bbappend

```diff
@@ -7,4 +7,10 @@ FILESEXTRAPATHS_append := ":${THISDIR}/files"
 SRC_URI_append_imx8m-var-dart = " \
 	file://imx8m-var-dart-integrate-balena.patch \
 	file://0001-imx8m_var_dart.h-Set-ads7846-chipselect-before-imx_s.patch \
+	file://0001-Add-support-for-OS_KERNEL_CMDLINE.patch \
 "
+
+do_generate_resin_uboot_configuration_append() {
+    fl = open(os.path.join(d.getVar('S'), 'include', 'config_resin.h'), 'a')
+    fl.write("#define %s %s\n" % ('OS_KERNEL_CMDLINE', d.getVar('OS_KERNEL_CMDLINE')))
+}
```

Comment:
meta-balena 2.35 included os_cmdline support in it so the way this should be done is the following: https://github.com/balena-os/balena-beaglebone/pull/189/commits/995dc55af3d8a5fcb7a898e507d54e14956d27dc

Reply:
Thank you @floion I've updated it